buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/JUnitXmlTestEventsGenerator.groovy

```diff
@@ -96,10 +94,9 @@ class JUnitXmlTestEventsGenerator {
         testListener.afterSuite(testSuiteDescriptor.parent.parent, new DefaultTestResult(TestResult.ResultType.SUCCESS, 0, 0, 0, 0, 0, []))
     }
 
-    @groovy.transform.CompileStatic
-    // workaround for class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"
-    // using Jigsaw
-    AssertionError assertionError(/*must be Object*/Object failureText) { new AssertionError(failureText) }
+    private boolean notSkipped(JUnitTestCase testCase) {
+        return testCase.skipped?.toString() == null
```

Comment:
How is that different to `testCase.skipped == null`?

Reply:
Seems no difference. Fixed.